pkg/cloudprovider/provider/kubevirt/provider.go

```diff
@@ -373,6 +374,10 @@ func (p *provider) Create(machine *clusterv1alpha1.Machine, data *cloudprovidert
 
 	resourceRequirements := kubevirtv1.ResourceRequirements{}
 	labels := map[string]string{"kubevirt.io/vm": machine.Name}
+	// Add a common label to all VirtualMachines spawned by the same MachineDeployment
+	if mdLabel, ok := machine.Labels["machine"]; ok {
+		labels["machine"] = mdLabel
```

Comment:
What is the point here? I mean why you are adding this label on the vm ?

Reply:
To have a common label among VMs to all VMs created from the same MachineDeployment.
Needed for example for the affinity we discussed yesterday.

Reply:
Yeah but the label that you are adding, isn't the machine deployment name :)

Reply:
@moath, as discussed, updated with the MachineDeployment name, rather than the `machine` label using the helper function.
Please review